src/main/java/com/project/videotecha/model/Theater.java

```diff
@@ -0,0 +1,50 @@
+package com.project.videotecha.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "theater")
+public class Theater {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private Integer capacity;
+
+    public Theater() {
+    }
+
+    public Theater(Long id, String name, Integer capacity) {
+        this.id = id;
+        this.name = name;
+        this.capacity = capacity;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
You don't need setter here since we'll only used predefined values with whom you already populated database.